terraform/k8s_board.tf

```diff
@@ -0,0 +1,41 @@
+data "honeycombio_query_specification" "k8s-metrics-query" {
+  calculation {
+    op     = "AVG"
+    column = "metrics.cpu.usage"
+  }
+
+  calculation {
+    op     = "AVG"
+    column = "metrics.memory.usage"
+  }
+
+  filter {
+    column = "k8s.resource.type"
+    op     = "="
+    value = "node"
+  }
+
+  breakdowns = ["k8s.node.name"]
+  orders     = [
```

Comment:
This should be:

```hcl
  order {
      column = "metrics.cpu.usage"
      op         = "AVG"
      order    = "descending"
    }
```

Reply:
Gotcha! I'll add that to the GH actions flow as well 👍